spec/features/arclight_spec.rb

```diff
@@ -84,6 +84,11 @@
           expect(page).to have_css('h3 a', text: 'Place')
           expect(page).to have_css('li .facet-label', text: 'Mindanao Island (Philippines)', visible: false)
         end
+
+        within('.blacklight-access_subjects_sim') do
+          expect(page).to have_css('h3 a', text: 'Subject')
+          expect(page).to have_css('li .facet-label', text: 'Fraternizing', visible: false)
```

Comment:
Hi Darren - I'm confused because I think all four are there. Those are the only ones issue #11 specifies. Am I missing your point?
<controlaccess>
      <subject source="mesh">Societies</subject>
      <function source="local">Fraternizing</function>
      <genreform source="aat">Photographs</genreform>
      <occupation source="mesh">Medicine</occupation>
      <corpname role="fmo" source="ingest">Alpha Omega Alpha</corpname>
      <persname source="mesh">Root, William Webster, 1867-1932</persname>
      <persname source="mesh">Bierring, Walter L. (Walter Lawrence), 1868-1961</persname>
      <geogname source="lcsh">Mindanao Island (Philippines)</geogname>
    </controlaccess>